cluster/examples/kubernetes/rook-operator.yaml

```diff
@@ -195,6 +195,10 @@ spec:
         # Set the path where the Rook agent can find the flex volumes
         # - name: FLEXVOLUME_DIR_PATH
         #  value: "<PathToFlexVolumes>"
+        # Allow rook to create multiple file systems. Note: This is considered
+        # an experimental feature in Ceph and should be enabled at your own risk.
```

Comment:
could we include a link in this comment to the ceph docs explaining what the risk is?

Reply:
Done.